src/common/fail_loc.c

```diff
@@ -35,7 +35,7 @@ uint64_t daos_fail_value;
 uint64_t daos_fail_num;
 
 void
-daos_reset_fail_loc()
+daos_fail_loc_reset()
```

Comment:
(void) parameter list?  I guess it's not required since you have it in the prototype